boredGame/Player.swift

```diff
@@ -35,8 +36,9 @@ func movePlayer(_ player: SKSpriteNode, _ ground: Ground, _ direction: playerDir
         }
     case .left:
         if (player.position.x - moveAmount) < leftMostX {
-            let movePlayerLess = SKAction.moveBy(x: -(player.position.x - leftMostX), y: 0, duration: moveDuration)
```

Comment:
This was not supposed to move the ground backwards. It'll actually just stop the guy from running left. If his body is going to move farther than the leftmost, we will place him at the leftmost position. Hopefully restoring this will prevent him from running off!